ui/src/App.tsx

```diff
@@ -99,21 +99,28 @@ const App = () => {
   const [modalEndTime, setModalEndTime] = useState<string>(DEFAULT_END_TIME);
   const [modalAllDay, setModalAllDay] = useState<boolean>(false);
   const [apiError, setApiError] = useState<boolean>(false);
+  const [rangeStart, setRangeStart] = useState<string>("");
+  const [rangeEnd, setRangeEnd] = useState<string>("");
+
+  const loadEventsForDateRange = (e: any) => {
```

Comment:
can we change this function to be named what it actually does, something like `setRange`? Alternatively (I think I'd prefer) to just have these state updates happen inline where `loadEventsForDateRange` is being called in the `datesSet` callback
```
  datesSet={({ startStr, endStr }) => {
    setRangeStart(new Date(startStr).toISOString());
    setRangeEnd(new Date(endStr).toISOString());
  }}
```

I'm also curious if `startStr` is already ISO, we don't have to flip it back and forth as a Date and then a string again. What format is `startStr` and `endStr`?

Reply:
It's not already ISO, it's in some other relative date format. So, I believe we need to do this transformation here.

Original string: 2023-01-01T00:00:00-08:00
ISO string: 2023-01-01T08:00:00.000Z